app/lib/theme/AnticipatedFundingPerFiscalYearArrayFieldTemplate.tsx

```diff
@@ -0,0 +1,64 @@
+import { ArrayFieldTemplateProps } from "@rjsf/core";
+import NumberFormat from "react-number-format";
+
+const AnticipatedFundingPerFiscalYearArrayFieldTemplate = (
+  props: ArrayFieldTemplateProps
+) => {
+  const { formContext, formData } = props;
+  console.log("formdata", formData);
+
+  if (!formContext.anticipatedFundingArray) {
+    return <div>No array, ask the designers what they want here</div>;
+  }
+
+  return (
+    <div>
+      {formContext.anticipatedFundingArray.edges.map(({ node }) => {
+        console.log("node", node);
+        return (
+          <>
+            {node && node.fiscalYear && (
```

Comment:
If there's no fiscal year (the user hasn't filled out any of date sent to csnr, submitted date, or due date), then the amount won't appear in any of the anticipated funding fields. I don't think it can work any other way, because without having a date we don't know where to add it, but it could result in a missing milestone armount.